kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeJobStatus.js

```diff
@@ -46,8 +44,6 @@ define([
     JobLogViewer,
     JobStatusTableTemplate,
     HeaderTemplate,
-    LogPanelTemplate,
```

Comment:
These were not used in this file (or anywhere in the codebase), so I removed the declarations and deleted the files.